krux_boto/boto.py

```diff
@@ -25,130 +25,152 @@
 # Internal libraries
 #
 
-from krux.logging       import get_logger, LEVELS, DEFAULT_LOG_LEVEL
-from krux.stats         import get_stats
-from krux.cli           import get_parser, get_group
+from krux.logging import get_logger, LEVELS, DEFAULT_LOG_LEVEL
+from krux.stats import get_stats
+from krux.cli import get_parser, get_group
+
 
+# Constants
+ACCESS_KEY = 'AWS_ACCESS_KEY_ID'
+SECRET_KEY = 'AWS_SECRET_ACCESS_KEY'
+NAME = 'krux-boto'
 
-### Constants
-ACCESS_KEY      = 'AWS_ACCESS_KEY_ID'
-SECRET_KEY      = 'AWS_SECRET_ACCESS_KEY'
-NAME            = 'krux-boto'
-DEFAULT_REGION  = 'us-east-1'
+# Defaults
+DEFAULT = {
+    'log_level': lambda: DEFAULT_LOG_LEVEL,
```

Comment:
for the more novices pythonistas (i.e., me ;) - can you add a comment why 'lambda' and not just the value?